pages/disability/eligibility/hazardous-materials-exposure/agent-orange/c-123-aircraft.md

```diff
@@ -75,19 +76,89 @@ You can upload supporting documents along with your online application, includin
   - Medical evidence (like a doctor’s report or medical test results)
 
 
-[Find out how to file a claim for disability compensation](/disability/how-to-file-claim/).<br>
+[Find out how to file a claim for disability compensation](/disability/how-to-file-claim/)<br>
 
 
 
+<li class="process-step list-four" markdown="0">
 
-### What if I need help filing my claim?
-Call the C-123 Hotline: <a href="tel:+18007498387">1-800-749-8387</a><br>
-<a href="mailto:VSCC123.VAVBASPL@va.gov">Email the St. Paul Regional Benefit Office</a>
+<p>Upload any supporting documents along with your online application, including:</p>
+
+<div class="feature">
+
+<ul>
+  <li>Discharge or separation papers (DD214 or other separation documents)</li>
+  <li>USAF Form 2096 (unit where you were assigned at the time of the training action)</li>
+  <li>USAF Form 5 (aircraft flight duties)</li>
+  <li>USAF Form 781 (aircraft maintenance duties)</li>
+  <li>Dependency records (marriage certificate and children’s birth certificates)</li>
+  <li>Medical evidence (like a doctor’s report or medical test results)</li>
+</ul>
+
+</div>
+
+</li>
+</ol>
```

Comment:
This page fails our accessibility tests -

```
Failed [fail]: (http://localhost:3001/disability/eligibility/hazardous-materials-exposure/agent-orange/c-123-aircraft/: [serious] <li> elements must be contained in a <ul> or <ol>
```

@bethpotts @DanielleSoCompany Can you take a look at this? I think the floating `</ol>` on line 101 and the `<li class="process-step list-four" markdown="0">` can be removed, but I'm not sure.

Reply:
@ncksllvn I think I fixed it. Looked like some random content got thrown in somehow. I removed it. Let me know if all looks good now.

Reply:
I think the errant content may have been meant for the online application, not this static page.